base/markdown/parse/util.jl

```diff
@@ -93,7 +93,7 @@ function startswith{T<:String}(stream::IO, ss::Vector{T}; kws...)
 end
 
 function startswith(stream::IO, r::Regex; eat = true, padding = false)
-  @assert beginswith(r.pattern, "^")
+  @assert Base.startswith(r.pattern, "^")
```

Comment:
The `Markdown` module has already defined a function named `startswith`, so the name will be overridden without `Base.`.